h/migrations/versions/9e6b4f70f588_removing_trailing_from_pdf_urns.py

```diff
@@ -8,16 +8,16 @@
 # pylint: disable=invalid-name, wrong-import-position
 import logging
 
+import sqlalchemy as sa
+from alembic import op
+from h.db import types
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import sessionmaker
+
```

Comment:
I thought isort knew about first party libraries and separated them out? Has this not be configured?

Reply:
I think either the diff is confusing or GitHub got confused here---if you open the file directly, it has actually grouped them right.

Reply:
See: https://github.com/hypothesis/h/blob/isort/h/migrations/versions/9e6b4f70f588_removing_trailing_from_pdf_urns.py